### PR TITLE
[FEAT] Show Boosted Time/Sell Price

### PR DESCRIPTION
**Pull request by j923sneaks**
# Description

This PR covers the ff:
- display reduced time on `Seeds.tsx` and `InventoryTabContent.tsx`
- Apply green highlight when boosted
- Rename `pricing.ts` to `boosts.ts`

One thing I can't seem to fix is at `Seeds.tsx` i.e cannot shrink flex item content width when reduced time is multiline-ed. CSS is not my strongest suit 😅 

sfl-mid-string-multiline-content-width

Fixes #issue N/A

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

### Manual Testing 
Local state with edited `INITIAL_FARM` inventory

Seeds modal - reduced plant time
sfl-time-boost-1
sfl-time-boost-2

Inventory modal - reduced plant time
sfl-time-boost-3

Crops modal - increased sell price
sfl-sell-boost

Regression - No Scarecrow NFT
sfl-regression-no-boost

# Checklist:

- [ ] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by adamhannigan** [comment]
If the aim is to solve the confusion around the boosted crop times + prices, I feel we will need to add more than green text to hint to the player the change. I always take the approach that we are designing it for non-tech savvy people, we need to dumb it down as much as possible 😅 

I would suggest adding a boost icon (usually a lightning bolt). In the future we could also add a tooltip showing the skill/item boosts?

I remember NIGHT did a cool badge which we could use to represent skill + NFT boosts 

https://github.com/sunflower-land/sunflower-land/issues/398#issuecomment-1073050191

What do you think?

**Comment by j923sneaks** [comment]
@adamhannigan I actually wanted to that but I didn't have the asset (and was not following art submissions) 😅 

I cannot chat at artists-chat atm (related to the ticket i raised), but hey @ItsNight may I have the lightning bolt icon image source? the one linked in the badges ticket is for all badges (not sure if i can just crop that...

**Comment by j923sneaks** [comment]
Added lightning icon beside timer when boosted

icon-1
icon-2
icon-3

**Comment by adamhannigan** [comment]
@j923sneaks Looks better to me 😄 

Just a nitpick: I would suggest removing the green text though. It doesn't fit the brown UI panel 😅

**Comment by j923sneaks** [comment]
removed green text

image
image
image

**Review by adamhannigan — APPROVED** [review]
Great work! This will reduce a lot of the confusion around planting boosts 👏
